src/components/Select/Select.js

```diff
@@ -11,14 +11,16 @@ class Select extends React.Component {
   static defaultProps = {
     disabled: false,
     error: null,
-    width: 100
+    width: 100,
+    helper: null 
```

Comment:
this should probably be an empty string since it's defined as a string in proptypes